packages/mobile/src/components/lineup-tile/LineupTileStats.tsx

```diff
@@ -158,6 +161,41 @@ export const LineupTileStats = ({
 
   const isReadonly = variant === 'readonly'
   const isScheduledRelease = isUnlisted && moment(releaseDate).isAfter(moment())
+
+  const renderLockedOrPlaysContent = () => {
```

Comment:
nit: i just now understood this name to mean "render Locked, or plays, content"
initially thought it mean "either render locked, or plays content"
hard to come up with better name tho.. maybe `renderLockedStatusOrPlayCount` or `renderLockedOrPlays` without the `Content`. Up to you!

Reply:
I agree with ^, the naming is a bit awkward. Maybe just `renderGatedOrUngatedContent`? I think its hard to name since its doing 3 different things

Reply:
yeah I copied the game from an existing component, but agree it's confusing. I can update it